app/controllers/application_controller.rb

```diff
@@ -46,6 +47,11 @@ def create_user_event(event_type, user = current_user)
 
   private
 
+  def disable_caching
+    response.headers['Cache-Control'] = 'no-store'
+    response.headers['Pragma'] = 'no-cache'
```

Comment:
These were the headers requested from the PenTest team. We can tweak the bits as necessary.